mantidproject/mantid
# Indirect simultaneous docs

**Description of work.**

The documentation for Indirect data analysis has been updated to match the new functionality introduced into the interface. Most notably Simultaneous fitting.

**To test:**
  * Build the docs and look at the Indirect Data Analysis documentation in the interfaces section. Check things make sense and that all the links work.



Fixes #27840  


*This does not require release notes* because documentation changes


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
